ephios/static/ephios/css/main.css

```diff
@@ -102,3 +102,15 @@ This hack fixes that. Inspired by https://github.com/select2/select2/issues/4220
     /*border: 1px solid #000000;*/
     border-radius: 50%;
 }
+
+.table-fixed {
+    table-layout: fixed;
+}
+
+.calendar-row-height {
+    height: 100px;
+}
+
+.today {
+    background-color: #cbcbcb;
+}
```

Comment:
This should probably be in a calendar-specific css file.

Reply:
I think `.today` does not make clear that it is used for a calendar and it might clash with other css in the future.
 Either specify the selectors or move to a specific css file.